### ISSUE TITLE
[Bug Report] Should remove keyspace and shard from TabletPicker. 

### ISSUE THREAD

**Issue by gerayking** [wesql/wescale#305]
### Overview of the Issue

Currently, vttablet uses TablePicker to select a serving tablet during the execution of onlineDDL. This selection is based on candidates identified via etcd. However, the TabletPicker relies on keyspace and shard to locate a serving tablet which recored in etcd. We need to modify TabletPicker to ensure the accurate execution of onlineDDL.
code of TablePicker:
```go
// TabletPicker gives a simplified API for picking tablets.
type TabletPicker struct {
	ts          *topo.Server
	cells       []string
	keyspace    string
	shard       string
	tabletTypes []topodatapb.TabletType
	inOrder     bool
}
```

### Reproduction Steps

nil

### Binary Version

```sh
Version: 16.0.0 (Git revision c98e99592add630a6cfebbf540c9e79c3a1d9770 branch 'test/add_onlineddl_scheduler_test') built on Thu Sep 21 15:05:23 CST 2023 by geray@bogon using go1.20.4 darwin/arm64
```

### Operating System and Environment details

```sh
darwin/arm64
```

**Comment by earayu** [comment]
It seems similar to Health Check. During Health Check, keyspace is default to `sidecardb.SidecarDbName`

**Comment by gerayking** [comment]
> It seems similar to Health Check. During Health Check, keyspace is default to `sidecardb.SidecarDbName`

Yes, I think there are two potential solutions to address this issue:

+ Rewrite the Pick function in the TabletPicker to accommodate the current requirements for selecting the appropriate serving tablet.
+ Set the default keyspace to sidecardb.SidecarDbName to align with the expected behavior.

**Comment by gerayking** [comment]
fixed by #320
